test-app/tests/integration/components/input-field-test.gts

```diff
@@ -21,7 +21,7 @@ module('Integration | Component | InputField', function (hooks) {
 
     const label = '[data-label]'
     const input = '[data-input]';
-    const inputId = find(input)?.getAttribute('id') || ''; 
```

Comment:
Prettier picked up on a bunch of changes, but I removed them and only applied the empty spacing changes here

Reply:
Thanks for this! Um...how did you selectively apply the prettier changes??

Reply:
Hm, so `prettier --check` doesn't run in CI then it seems?

Reply:
This is super interesting.  So `prettier --check` is returning as all is good.  Running `pnpm format`, I see the following:

```bash
➜  test-app git:(fix-input-error-outline) ✗ pnpm format

> test-app@0.0.0 format /workspace/ember-toucan-core/test-app
> prettier -w .

.eslintrc.cjs 23ms
.prettierrc.js 5ms
.template-lintrc.js 3ms
.watchmanconfig 1ms
app/app.ts 79ms
app/config/environment.d.ts 6ms
app/index.html 16ms
app/router.ts 3ms
app/styles/app.css 7ms
app/templates/application.hbs 17ms
config/ember-cli-update.json 2ms
config/ember-try.js 5ms
config/environment.js 4ms
config/optional-features.json 1ms
config/targets.js 2ms
ember-cli-build.js 2ms
package.json 2ms
README.md 24ms
testem.js 3ms
tests/helpers/index.ts 4ms
tests/index.html 4ms
tests/test-helper.ts 2ms
tsconfig.json 1ms
types/global.d.ts 2ms
types/test-app/index.d.ts 2ms
```

Which is interesting, as prettier seems to be skipping over `.gjs/gts` files.  So it seems like in VSCode, prettier auto formats things as you'd expect, but these files are getting missed via the CLI.  Any ideas?

Nothing really sticks out to me in the [prettierrc file](https://github.com/CrowdStrike/ember-toucan-core/blob/main/test-app/.prettierrc.js).  We have an override for `.gjs/gts` files, but I can't seem to get any of [these rules](https://prettier.io/docs/en/options.html) to run against this extension.